include/aligator/modelling/dynamics/wheeled-inverted-pendulum.hpp

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include <math.h>
+#include "aligator/core/vector-space.hpp"
+
+using T = double;
+using VEC = aligator::VectorSpaceTpl<T>;
+
+using namespace aligator;
+ALIGATOR_DYNAMIC_TYPEDEFS(T);
```

Comment:
We have to re-do this entire part.
- you can't use-declare the entire namespace in a public header
- you can't add the typedefs to the global namespace
- you imported the wrong headers